server/bundles/io.cloudbeaver.model/src/io/cloudbeaver/model/session/WebSession.java

```diff
@@ -429,6 +431,16 @@ private void resetSessionCache() throws DBCException {
         }
     }
 
+    private void resetTempFolder(){
+        Path path = getWorkspace().getAbsolutePath().resolve("temp-sql-upload-files").resolve(this.getSessionId());
+        try {
+            Files.delete(path);
```

Comment:
As far as I remember, this method will throw an error if the folder is not empty

Reply:
You can use `IOUtils.deleteDirectory`

Reply:
and may be it better do it via event